data/Sri Dasam Granth/1389.json

```diff
@@ -592,7 +592,7 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "The Lord of the Universe, Knows all the sciences and breaks the knots of all complications.11.",
+              "translation": "and breaks the knots of all complications.11.",
```

Comment:
Capitalise?

Reply:
Probably part of the line before it, not sure whether we should in case we combine two lines. Does previous translation end i with a punctuation mark

Reply:
Probably part of the line before it, not sure whether we should in case we combine two lines. Does previous translation end i with a punctuation mark

Reply:
Ditto second point

Reply:
@bhajneet it does not, but some do end with commas etc